roles/percona-server/tasks/serverspec.yml

```diff
@@ -8,3 +8,5 @@
             dest=/etc/serverspec/spec/localhost/
             mode=0755
   with_fileglob: ../templates/etc/serverspec/*
+  notify:
+    - restart mysql server
```

Comment:
why do we need to notify restart mysql? We are just writing our the checks. It is triggered by sensu client.